VerifyMe/verify_me.py

```diff
@@ -10,9 +10,16 @@
 from discord.ext import commands
 
 from addons.prefixed_cog import prefixed_cog
-from Miscellaneous.util import log, send_embeds
+from addons.embed_factory import EmbedFactory
+from Miscellaneous.util import log
 
 
+EMBED_GENERIC_ERROR = EmbedFactory.error()
+EMBED_MACID_ALREADY_REGISTERED = EmbedFactory.error(message=
+    "**This MacID is already registered to another Discord user.**\nPlease contact an Administrator if you believe this is a mistake.")
+EMBED_REQUEST_TIMED_OUT = EmbedFactory.error(message=
+    "**Your request has timed out**\nRestart the request by typing `~verify` in chat.")
+
```

Comment:
Storing the errors here instead of their own files may not be desirable, but it's up to you